projects/js-packages/components/components/zendesk-chat/test/component.tsx

```diff
@@ -1,39 +1,16 @@
 import { jest } from '@jest/globals';
 import { render, screen } from '@testing-library/react';
 import ZendeskChat from '../index';
-import utils from '../utils';
 
 describe( 'ZendeskChat', () => {
 	afterEach( () => {
 		// restore the spy created with spyOn
 		jest.restoreAllMocks();
 	} );
 
-	it( 'renders when Date and Time requirements are met', () => {
-		// Mock conditions in which widget should be shown
-		jest.spyOn( utils, 'isWithinAvailableChatTimes' ).mockReturnValue( true );
-		jest.spyOn( utils, 'isWithinAvailableChatDays' ).mockReturnValue( true );
-
+	it( 'renders the zendesk chat widget', () => {
```

Comment:
I am not even sure this needs a test anymore since we are just rendering an iframe, but I left it here because it was already written and I guess testing if it renders correctly is useful